idaes/models/properties/modular_properties/base/generic_property.py

```diff
@@ -1653,10 +1653,14 @@ def fix_initialization_states(self):
 
         # Also need to deactivate sum of mole fraction constraint
         for k in self.values():
-            try:
-                k.sum_mole_frac_out.deactivate()
-            except AttributeError:
-                pass
+            if not k.config.defined_state:
```

Comment:
Probably the thing to do here is check `if "flow_mol_phase_comp" not in b.define_state_vars()` like is done elsewhere, not nest a second try/except block in the first except block.